pkg/webhook/webhook.go

```diff
@@ -490,3 +511,49 @@ func (wh *webhook) validateCVC(ar *v1beta1.AdmissionReview) *v1beta1.AdmissionRe
 		"configured for operations other than UPDATE")
 	return response
 }
+
+// verifyExistenceOfSnapshots returns error when there are volumesnapshots or volumesnapshotDatas
+// related to volume and if any error occured else return nil
```

Comment:
verifyExistenceOfSnapshots returns error if  there are volumesnapshots or volumesnapshotData exists for a volume  or any API error in case.